README.md

```diff
@@ -66,65 +54,7 @@ $ docker run -v "$PWD":/workdir -e VFC_BACKENDS="libinterflop_mca.so" \
 
 ## Installation
```

Comment:
I would put this section first, and move previous docker section after with (maybe)  content in a sperate file.

Reply:
I put installation first. Since the docker section is really short, I think it's best to leave it in the main README.